.github/deps.repos

```diff
@@ -3,3 +3,11 @@ repositories:
     type: git
     url: https://github.com/Fields2Cover/Fields2Cover.git
     version: main
+  ortools_vendor:
+    type: git
+    url: https://github.com/Fields2Cover/ortools_vendor.git
+    version: main
+  navigation2:
```

Comment:
Lets not add in nav2, it causes the build to fail before we hit this work so I can't know if what you made compiles in an independent location. Plus this is a huge amount of code to compile

Reply:
Yeah just circling back to this.  The need to build ortools is a drag.  Takes ages.  

I just tweaked an alternative inside Fields2Cover, by @Gonzalo-Mier, that grabs a release build of ortools. 

I wonder if that mechanism should be moved into ortools_vendor.  Do you know if it is necessary for ros packages to be built from source, or is this usage of a tarball acceptable?

Reply:
I have done some more work on the Fields2Cover build.  

https://github.com/Fields2Cover/Fields2Cover/pull/133
https://github.com/Fields2Cover/Fields2Cover/pull/133/commits/f9f0a105d14bd16c850272c73c2cd735a3f75ca9